warehouse/templates/pages/help.html

```diff
@@ -158,6 +160,21 @@ <h3 id="verified-email">{{ verified_email() }}</h3>
         <p>
           You can manage your account's email addresses in your <a href="{{ request.route_path('manage.account') }}">Profile</a>. This also allows for sending a new confirmation email for users who signed up in the past, before we began enforcing this policy.
         </p>
+
+
+        <h3 id="compromised-password">{{ compromised_password() }}</h3>
+          <p>
+            Each time a user supplies a password &mdash; while registering, authenticating, or updating their password &mdash; PyPI securely checks whether that password has appeard in public data breaches.
```

Comment:
Would it be a good idea to lead off with the fact that this does not mean PyPI itself was breached?

Reply:
"PyPI itself has not suffered a breach, this is a protective measure to reduce the risk of [credential stuffing](https://www.owasp.org/index.php/Credential_stuffing) attacks against PyPI and it's users."

Reply:
Typo in "whether that password has appeard" -- should be "appeared".

Reply:
something like that? not sure if there's a better way to word "PyPI itself has not suffered a breach"

Reply:
That sounds like a good idea to me.